src/app-logic/constants.js

```diff
@@ -41,6 +41,11 @@ export const TRACK_MEMORY_HEIGHT =
 export const TRACK_MEMORY_LINE_WIDTH = 2;
 export const TRACK_MEMORY_COLOR = 'orange';
 
+// The following values are for the bandwidth track.
+export const TRACK_BANDWIDTH_HEIGHT = 25;
```

Comment:
The graph still looks OK (IMO, but others might disagree) if we reduce this to 20px. I'm not sure it's a good idea though, as that would create a new track size, and consistency might be more valuable than the 5px we could save.